lib/utilities.js

```diff
@@ -12,8 +12,8 @@ function camelCase(string) {
     throw new TypeError('First argument must be a string');
   }
 
-  // no hyphen found
-  if (string.indexOf('-') === -1) {
+  // customStyle or no hyphen found
+  if (string.indexOf('--') === 0 || string.indexOf('-') === -1) {
```

Comment:
As I [asked](https://github.com/remarkablemark/html-react-parser/pull/82#discussion_r241745486) in #82, do you think it's useful making the declaration value check strict here?

```diff
-var CUSTOM_PROPERTY_OR_NO_HYPHEN_REGEX = /^--[a-zA-Z]+$|^[^-]+$/;
+var CUSTOM_PROPERTY_OR_NO_HYPHEN_REGEX = /^--[a-zA-Z-]+$|^[^-]+$/;
```

Reply:
your modified regex seems to work perfectly, so replaced the strict check with your regex matcher. I also added the tests and modified the commit message as preferred. 

Thank you for your help !